model.js

```diff
@@ -45,7 +45,7 @@ selectCommentsByArticleId = async (article_id) => {
     if (!comments) {
         return Promise.reject({
```

Comment:
we also shouldn't have two rejects for the same thing - you need to either reject in your model OR in your controller at the moment your reject inside the model means that the reject in the controller is never reached - whichever one you choose to do you should be consistent with this. Usually we reject in the model so that the controller then invokes next with the rejected promises in the catch block